buildkitd/buildkitd.go

```diff
@@ -212,12 +214,16 @@ func Start(ctx context.Context, image string, settings Settings, reset bool) err
 	if err != nil {
 		return err
 	}
-	// Pulling is not strictly needed, but it helps display some progress status to the user in
-	// case the image is not available locally.
-	err = MaybePull(ctx, image)
-	if err != nil {
-		fmt.Printf("Error: %s. Attempting to start buildkitd anyway...\n", err.Error())
-		// Keep going - it might still work.
+	if reportPull {
```

Comment:
Why are we gating this right now?

Reply:
Not sure if this is causing user drops - I would turn it off for now as an extra precaution.